tests/test_transforms.py

```diff
@@ -191,8 +196,8 @@ def test_calculate_voltage_from_gmr__returns_correct_values():
     )
     test_data = np.vstack((np.zeros(3), test_data))
     original_test_data = copy.deepcopy(test_data)
-    reference_voltage = 3.3
-    adc_gain = 32
+    reference_voltage = 2.5
```

Comment:
these values show up in both the code and the tests, so it may be worth refactoring them to a single source of truth as constants somewhere